### ISSUE TITLE
Cursor disappeared and gets stuck on dice select

### ISSUE THREAD

**Issue by DocSox** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#440]
The cursor disappeared during play as a GM and would not reappear.  Noticed Dice were being selected but could not disengage the dice selection or roll the dice.  Had to quite and reopen to fix.

Was unable to reproduce error but a player had it happen to him 3 times during play as well.

**Comment by LuPro** [comment]
I've seen similar behavior in #29, did the mouse cursor not come back for you upon opening the exit menu? (Alt + F4 or I think pressing Esc should open it as well)

**Comment by DocSox** [comment]
No, nothing I did would get the cursor to come back and none of the
commands would work to bring up any menus or change modes either.  Esc
would not bring up the exit menu but I did not try alt-F4.  The only
command that would work was [space] to hide and bring back the UI.  I even
tried alt tab to see if changing programs would work and the cursor would
come back when not in Talespire but would return to the dice select only
once Talespire was re-selected.

**Comment by ryanmurphyfilm** [comment]
This is happening to me as well. Right now it happens almost every time I try and move a character who is on the first floor of a building with more than one floors. Trying to exit the game fixes the issue.

**Comment by brcoding** [comment]
This appears for me when I am dragging a character and i let go when over a button, or dice. It appears the mouse up doesn't honor the dragging state.

**Comment by MrMonop** [comment]
When my mouse disappears, I find if I alt-tab out of the game, hold Alt to see Talespire's window and click to "close" , Talespire pops up with an "are you sure you want to exit?" window which makes my mouse reappear.  I select "No" and am able to move correctly from there.

**Comment by USMCM1A1** [comment]
Two of my players experienced similar behavior: mouse pointer wiggled around crazy, disappeared, and were unable to select anything.  _When I (DM) left the game, they regained mouse control_.

**Comment by MrMonop** [comment]
I found a temporary solution for our group.  When you lose the mouse
cursor, Alt+tab out but hold down Alt so you see all windows and can
select.  Click on the “X” to close TaleSpire and the “Are you sure you want
to close?” Dialogue will pop up. At that point click on there TaleSpire
window and your cursor will re-appear to be able to select the yes or no
option.  Select “No” and you’ll be back in the game and everything should
be functioning normally.

**Comment by USMCM1A1** [comment]
Alt-f4 is working for my players to regain mouse control.

**Comment by MrMonop** [comment]
Confirmed, achieves the same result with less steps lol thanks!
